packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -941,8 +941,12 @@ export class PipelineEditor extends React.Component<
           } else {
             // in this case, pipeline was last edited in a "old" version of Elyra and
             // it needs to be updated/migrated.
-            showDialog({
-              title: 'Migrate pipeline?',
+            const filePath = this.widgetContext.path;
+            const extension = path.extname(filePath);
+            const pipelineName = path.basename(filePath, extension);
```

Comment:
This can all be one line since you only use `pipelineName` below

Reply:
I thought it would be a long line and not as readable:
`const pipelineName = path.basename(this.widgetContext.path, path.extname(this.widgetContext.path));`
If that's still ok I can make the change

Reply:
the linter will line wrap it and it'll be readable, that's what every other call to `path.basename` in our code looks like.

Reply:
though you could also do
```suggestion
            const filePath = this.widgetContext.path;
            const pipelineName = path.basename(filePath, path.extname(filePath));
```
instead, which would also be good since you call `this.widgetContext.path` below and that could then use `filePath` as well